pkg/insights/metrics.go

```diff
@@ -57,7 +57,7 @@ func (c *InsightsRecommendationCollector) Collect(ch chan<- prometheus.Metric) {
 			prometheus.NewDesc(c.metricName, "", []string{}, prometheus.Labels{
 				"description": rec.Description,
 				"total_risk":  totalRiskToStr(rec.TotalRisk),
-				"info_link":   fmt.Sprintf("https://console.redhat.com/openshift/insights/advisor/recommendations/%s%%7C%s", ruleIDStr, rec.ErrorKey),
+				"info_link":   fmt.Sprintf("https://console.redhat.com/openshift/insights/advisor/clusters/%s?first=%s%%|%s", rec.ClusterID, ruleIDStr, rec.ErrorKey),
```

Comment:
Hmm I am just wondering what the `first` query parameter is for

Reply:
I think it has something to do with order of recommendations in the hybrid cloud console, but I am not 100% sure. If you put for example `?second=` it works as well. It works even if you leave just `?` in the url and delete the `first=` part. But default url contains `?first=` if you get to it through hybrid cloud console site so I would just let it be.

Reply:
OK thanks for checking. I would like to keep it as simple as possible so having no param would be the best IMO but we can ask someone to tell us more about this.